.github/workflows/update-completed-sprint-on-issue-closed.yml

```diff
@@ -7,8 +7,19 @@ on:
     types: [closed]
 
 jobs:
-  update_completed_sprint:
-    uses: paulbellamy/stellar-actions/.github/workflows/update-completed-sprint-on-issue-closed.yml@update-completed-sprint-on-issue-closed
-    with:
-      project_name: "Platform Scrum"
-      field_name: "CompletedSprint"
+  update-completed-sprint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
```

Comment:
Is it worth pinning to a specific sha to avoid code changing without us knowing?

Reply:
Let's do that if it breaks

Reply:
The risk I was thinking of was compromised action or developer's account. With that somebody could publish malicious action that steals creds or repo data.
It's not very frequent problem and in this particular case our exposure would be limited to access to projects so it maybe acceptable.

But getting into habit of using sha everywhere might be good from security perspective.